documentation/docs/20-core-concepts/10-routing.md

```diff
@@ -12,6 +12,19 @@ At the heart of SvelteKit is a _filesystem-based router_. The routes of your app
 
 Each route directory contains one or more _route files_, which can be identified by their `+` prefix.
 
+Summary of the routing files:
+
+| File name         | Role                                                                | Propagates to sub-dirs | SSR | CSR |
```

Comment:
SSR and CSR don't make sense for the API endpoints. You can invoke endpoints while doing server side rendering or client-side rendering. I'm guessing you're referring to the endpoint code itself running on either the client or server
```suggestion
| File name         | Role                                                                | Propagates to sub-dirs | Runs on server | Runs on client |
```